### PR TITLE
Feat/lesson02

### PR DESCRIPTION
**Pull request by lewtun**
This PR contains the notebooks and images for lesson 02. I will add the badges later once this is merged to master.

### CODE REVIEW

File: notebooks/00_core.ipynb [hunk 1]
```diff
@@ -27,7 +27,8 @@
     "# export\n",
```
Comment (lvwerra) [inline notebooks/00_core.ipynb]: <p>why did you add subprocess?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

File: notebooks/lesson02_exploratory-data-analysis.ipynb [hunk 2]
```diff
@@ -0,0 +1,2971 @@
+{
```
Comment (lvwerra) [inline notebooks/lesson02_exploratory-data-analysis.ipynb:1]: <p>Since this shows california it would be cool to see the actual map next to it or have them look it up and explain the hotspots? just an idea.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Comment (lvwerra) [inline notebooks/lesson02_exploratory-data-analysis.ipynb:1]: <p>maybe a comment about duplicates could be useful. in the example you give you have a lot of duplicates in the left table and it might not be straightforward what happens if you then left join. no need to go through all combinations but raise awareness how it works.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Comment (lvwerra) [inline notebooks/lesson02_exploratory-data-analysis.ipynb:1]: <p>I'd add a remark about the choice of axis since it happened countless times to me that I dropped columns instead of rows.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Comment (lvwerra) [inline notebooks/lesson02_exploratory-data-analysis.ipynb:1]: <p>we've never really introduced the index concept in pandas. maybe we should add a comment somewhere what it is and that it can also be other numbers than 0, 1, ..., n and even objects?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='84'/>

File: notebooks/00_core.ipynb (hunk 1 above)
Comment (lewtun) [reply notebooks/00_core.ipynb]: <p>Ah yes, this is an artifact from a hack I was trying to implement that would skip the download if the file already existed. I'll remove it for now</p>

File: notebooks/lesson02_exploratory-data-analysis.ipynb (hunk 2 above)
Comment (lewtun) [reply notebooks/lesson02_exploratory-data-analysis.ipynb:1]: <p>Great idea - I'll add it in the next commit</p>

Comment (lewtun) [reply notebooks/lesson02_exploratory-data-analysis.ipynb:1]: <p>Good catch - i'll add something </p>

Comment (lewtun) [reply notebooks/lesson02_exploratory-data-analysis.ipynb:1]: <p>Done</p>

Comment (lewtun) [reply notebooks/lesson02_exploratory-data-analysis.ipynb:1]: <p>done</p>
